.github/CODEOWNERS

```diff
@@ -1407,7 +1399,7 @@ spec/lib/common/client/configuration @department-of-veterans-affairs/backend-rev
 spec/lib/common/client/middleware @department-of-veterans-affairs/backend-review-group
 spec/lib/common/client/session_spec.rb @department-of-veterans-affairs/backend-review-group
 spec/lib/common/convert_to_pdf_spec.rb @department-of-veterans-affairs/backend-review-group
-spec/lib/common/exceptions/detailed_schema_errors_spec.rb @department-of-veterans-affairs/backend-review-group
+spec/lib/common/exceptions/ @department-of-veterans-affairs/backend-review-group
```

Comment:
Do no other teams own any of the exceptions? Or, does it make sense for us to own them all?

Reply:
Yeah, I think I've convinced myself that it's fine for us to own all of these. Or, at least it shouldn't block this PR since another team is waiting on it.